### ISSUE TITLE
warning meesages valid?

### ISSUE THREAD

**Issue by bondo01** [Er0manga/Er0mangaDemo#6]
Getting this messages when starting up Er0manga

docker run --gpus all --rm -it -p 7860:7860 er0manga/er0mangademo
Detectron v2 is not installed
/home/user/Er0mangaSeg/mmseg/models/decode_heads/decode_head.py:120: UserWarning: For binary segmentation, we suggest using`out_channels = 1` to define the outputchannels of segmentor, and use `threshold`to convert `seg_logits` into a predictionapplying a threshold
  warnings.warn('For binary segmentation, we suggest using'
/home/user/Er0mangaSeg/mmseg/models/losses/cross_entropy_loss.py:282: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
Loads checkpoint by local backend from path: /home/user/Er0mangaSeg/pretrained/convnext_1024_iter_400.pth
Segmentation initialized
Inpainting initialized
Running on local URL:  http://0.0.0.0:7860
